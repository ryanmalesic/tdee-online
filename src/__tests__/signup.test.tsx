import React from 'react';
import renderer from 'react-test-renderer';

import SignUp from '../pages/signup';

it('renders signup unchanged', () => {
  const tree = renderer.create(<SignUp />).toJSON();
  expect(tree).toMatchSnapshot();
});
