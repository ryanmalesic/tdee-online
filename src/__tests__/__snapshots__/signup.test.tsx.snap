// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders signup unchanged 1`] = `
<section
  className="hero is-fluid is-fullheight background"
>
  <div
    className="hero-head heroHead"
  >
    <nav
      className="navbar"
      role="navigation"
    >
      <div
        className="navbar-brand"
      >
        <a
          className="navbar-item"
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <h1
            className="title"
          >
            TDEE Online
          </h1>
        </a>
        <a
          className="navbar-burger burger "
          data-target="navbarBasicExample"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            aria-hidden="true"
          />
          <span
            aria-hidden="true"
          />
          <span
            aria-hidden="true"
          />
        </a>
      </div>
      <div
        className="navbar-menu"
        id="navbarBasicExample"
      >
        <div
          className="navbar-start"
        >
          <a
            className="navbar-item"
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Home
          </a>
        </div>
        <div
          className="navbar-end"
        >
          <div
            className="navbar-item"
          >
            <div
              className="buttons"
            >
              <a
                className="button has-background-primary-dark"
                href="/signup"
                onClick={[Function]}
                onMouseEnter={[Function]}
              >
                <strong>
                  Sign up
                </strong>
              </a>
              <a
                className="button has-text-dark has-background-grey-lighter"
                href="/signin"
                onClick={[Function]}
                onMouseEnter={[Function]}
              >
                Sign in
              </a>
            </div>
          </div>
        </div>
      </div>
    </nav>
  </div>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <div
        className="columns is-centered is-mobile"
      >
        <div
          className="column is-full-mobile is-half-tablet"
        >
          <div
            className="box"
          >
            <h1
              className="title is-1"
            >
              Sign Up
            </h1>
            <h3
              className="subtitle is-3"
            >
              Start your journey.
            </h3>
            <form
              onSubmit={[Function]}
            >
              <div
                className="columns"
              >
                <div
                  className="column"
                >
                  <div
                    className="field"
                  >
                    <label
                      className="label"
                      htmlFor="firstName"
                    >
                      First Name
                    </label>
                    <div
                      className="control"
                    >
                      <input
                        autoComplete="given-name"
                        className="input"
                        id="firstName"
                        name="firstName"
                        onBlur={[Function]}
                        onChange={[Function]}
                        type="text"
                        value=""
                      />
                    </div>
                  </div>
                </div>
                <div
                  className="column"
                >
                  <div
                    className="field"
                  >
                    <label
                      className="label"
                      htmlFor="email"
                    >
                      Email
                    </label>
                    <div
                      className="control"
                    >
                      <input
                        autoComplete="username"
                        className="input"
                        id="email"
                        name="email"
                        onBlur={[Function]}
                        onChange={[Function]}
                        type="email"
                        value=""
                      />
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="field"
              >
                <label
                  className="label"
                  htmlFor="password"
                >
                  Password
                </label>
                <div
                  className="control"
                >
                  <input
                    autoComplete="new-password"
                    className="input"
                    id="password"
                    name="password"
                    onBlur={[Function]}
                    onChange={[Function]}
                    type="password"
                    value=""
                  />
                </div>
              </div>
              <div
                className="field"
              >
                <label
                  className="label"
                  htmlFor="confirmPassword"
                >
                  Confirm Password
                </label>
                <div
                  className="control"
                >
                  <input
                    autoComplete="new-password"
                    className="input"
                    id="confirmPassword"
                    name="confirmPassword"
                    onBlur={[Function]}
                    onChange={[Function]}
                    type="password"
                    value=""
                  />
                </div>
              </div>
              <div
                className="columns"
              >
                <div
                  className="column"
                >
                  <div
                    className="field"
                  >
                    <label
                      className="label"
                      htmlFor="birthdate"
                    >
                      Birthdate
                    </label>
                    <div
                      className="control"
                    >
                      <input
                        autoComplete="bday"
                        className="input"
                        id="birthdate"
                        name="birthdate"
                        onBlur={[Function]}
                        onChange={[Function]}
                        placeholder="yyyy-mm-dd"
                        type="date"
                        value=""
                      />
                    </div>
                  </div>
                </div>
                <div
                  className="column is-narrow"
                >
                  <div
                    className="field"
                  >
                    <label
                      className="label"
                      htmlFor="sex"
                    >
                      Sex
                    </label>
                    <div
                      className="control"
                    >
                      <div
                        className="select"
                      >
                        <select
                          autoComplete="sex"
                          id="sex"
                          name="sex"
                          onBlur={[Function]}
                          onChange={[Function]}
                          value=""
                        >
                          <option
                            value="MALE"
                          >
                            Male
                          </option>
                          <option
                            value="FEMALE"
                          >
                            Female
                          </option>
                        </select>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="field"
              >
                <div
                  className="columns is-variable is-0-mobile is-3-tablet"
                >
                  <div
                    className="column is-narrow"
                  >
                    <div
                      className="control"
                    >
                      <button
                        className="button has-background-primary-dark "
                        disabled={false}
                        type="submit"
                      >
                        Sign Up
                      </button>
                    </div>
                  </div>
                  <div
                    className="column"
                  >
                    <a
                      className="button is-link"
                      href="/signin"
                      onClick={[Function]}
                      onMouseEnter={[Function]}
                    >
                      Already a member? Sign In.
                    </a>
                  </div>
                </div>
              </div>
            </form>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hero-footer footer"
  >
    <span
      className="tag"
    >
      Photo by  
      <a
        href="https://unsplash.com/@brett_jordan?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
      >
        Brett Jordan
      </a>
       on 
      <a
        href="https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
      >
        Unsplash
      </a>
    </span>
  </div>
</section>
`;
